README.md

```diff
@@ -91,3 +91,32 @@ latest release branch.
 
 - [colmena](https://github.com/zhaofengli/colmena) -  A simple, stateless NixOS deployment tool
 - [robotnix](https://github.com/danielfullmer/robotnix) -  Build Android (AOSP) using Nix, used in their [CI](https://github.com/danielfullmer/robotnix/blob/38b80700ee4265c306dcfdcce45056e32ab2973f/.github/workflows/instantiate.yml#L18)
+
+## FAQ
+
+### nix-eval-jobs consumes too much memory / is too slow
+
+By default, nix-eval-jobs spawns as many worker processes as there are
+hardware threads in the system and limits the memory usage for each worker to
+4GB.
+
+However, keep in mind that each worker process may need to re-evaluate shared
+dependencies of the attributes, which can introduce some overhead for each
+evaluation or cause workers to exceed their memory limit. If you encounter
+these situations, you can tune the following options:
+
+`--workers`: This option allows you to set the number of evaluation workers that
+nix-eval-jobs should spawn. You can increase or decrease this number to
+optimize the evaluation speed and memory usage. For example, if you have a
+system with many CPU cores but limited memory, you may want to reduce the
+number of workers to avoid exceeding the memory limit.
+
+`--max-memory-size`: This option allows you to adjust the memory limit for each
```

Comment:
I think it may be worth mentioning that memory usage can spike higher than this, but that the worker will restart ASAP once the work has been done.